scripts/webpack/webpack.config.e2e.ts

```diff
@@ -40,7 +40,7 @@ const webpackConfig: webpack.Configuration = {
     ],
   },
   plugins: [
-    new webpack.DefinePlugin(getDefaultEnvironmentVars()),
+    new webpack.DefinePlugin(getDefaultEnvironmentVars(true)),
```

Comment:
As we run this build only in production this override is required, otherwise it's causing:

```
verb @fluentui/e2e test |  WARNING in DefinePlugin
verb @fluentui/e2e test |  Conflicting values for 'process.env.NODE_ENV'
verb @fluentui/e2e test |  '"development"' !== '"production"'
verb @fluentui/e2e test | 
```